README.md

```diff
@@ -4,6 +4,7 @@
 [![Public API](http://home.kk-sc.de:9000/api/badges/measure?key=com.minecolonies%3Aminecolonies%3Adevelop&blinking=true&metric=public_documented_api_density)](http://home.kk-sc.de:9000/overview?id=com.minecolonies%3Aminecolonies%3Adevelop)
 [![Lines of Code](http://home.kk-sc.de:9000/api/badges/measure?key=com.minecolonies%3Aminecolonies%3Adevelop&blinking=true&metric=ncloc)](http://home.kk-sc.de:9000/overview?id=com.minecolonies%3Aminecolonies%3Adevelop)
 [![Stories in Ready](https://badge.waffle.io/Minecolonies/minecolonies.png?label=help%20wanted&title=Help%20Wanted)](http://waffle.io/Minecolonies/minecolonies)
+[![BCH compliance](https://bettercodehub.com/edge/badge/inesgomes/minecolonies)](https://bettercodehub.com)

```

Comment:
can you change this to [![BCH compliance](https://bettercodehub.com/edge/badge/Minecolonies/minecolonies)](https://bettercodehub.com) `[![BCH compliance](https://bettercodehub.com/edge/badge/Minecolonies/minecolonies)](https://bettercodehub.com)` because That is a nice check we definitely can integrate :)